source/Halibut/DataStream.cs

```diff
@@ -149,12 +152,14 @@ public static DataStream FromStream(Stream source, Action<int> updateProgress)
             });
         }
         
+        [Obsolete]
         public static DataStream FromStream(Stream source, Action<int> updateProgress, Func<int, CancellationToken, Task> updateProgressAsync)
         {
             var streamer = new StreamingDataStream(source, updateProgress, updateProgressAsync);
             return new DataStream(source.Length, streamer.CopyAndReportProgress, streamer.CopyAndReportProgressAsync);
         }
 
+        [Obsolete]
         public static DataStream FromStream(Stream source)
```

Comment:
This is not `obsolete` streams have async methods that will be used when halibut is in async mode.